player/player.go

```diff
@@ -335,14 +340,37 @@ func (p *Player) ChangeClass(class string) {
   p.Save()
   cl := classes.Find(class)
   if cl == nil {
-    p.SendMsg("Couldn't find class '%s'. Please use `classes` to confirm the spelling.", strings.Title(class))
+    p.SendMsg(fmt.Sprintf("Couldn't find class '%s'. Please use `classes` to confirm the spelling.", strings.Title(class)))
+    return
+  }
+
+  if p.classIsLocked(cl.GetName()) {
```

Comment:
This logic should be in the change subclass method also. Eventually there will be Tier 3 classes and we wouldn't want players to be able to subclass a Tier 2 class they haven't unlocked (although maybe this isn't necessary if we have not-max-level protections in #ChangeSubclass)